src/controllers/imageController.js

```diff
@@ -9,7 +9,8 @@ const storage = multer.diskStorage({
     cb(null, DIR)
   },
   filename: function (req, file, cb) {
-    const uniqueSuffix = Date.now() + '-' + Math.round(Math.random() * 1E9) + path.extname(file.originalname)
+    let index = file.mimetype.indexOf("/")
+    const uniqueSuffix = Date.now() + '-' + Math.round(Math.random() * 1E9) + "." + file.mimetype.substring(index+1);
```

Comment:
I am not passing the file name anymore. Changed it to use memetype instead.

Reply:
Good thinking! Probably more secure that way